src/main.service.js

```diff
@@ -160,6 +160,28 @@ plugin.service('wgnConfigSrv', ['$q', '$firebase', 'znData', function ($q, $fire
 		return multi ? saveMulti(workspaceId, $ref, config) : saveSingle(workspaceId, $ref);
 	};
 
+	/**
+	 * Saves a plugin's secure configuration settings for the given workspace.
+	 *
+	 * @param {number} workspaceId
+	 * @param {String} configId The config id for multi configurations
+	 * @param {Object} config The configuration object to save.
+	 *
+	 * @return {Promise<*>}
+	 */
+	srv.saveSecure = function (workspaceId, configId, config) {
```

Comment:
maybe rename that 3rd parameter to `secureConfig` or `secureFieldData` or something. Took me a second to connect the dots.